File: tests/typesafety/python/flag_documented.py
# expect: passes
# title: The docstring says text or binary, and the checker cannot read it
def read_log(path: str, mode: str) -> int:
    """mode is "text" or "binary"."""
    record_size = 1 if mode == "text" else 8
    return len(path) * record_size


def tail() -> int:
    return read_log("app.log", "binry")

File: tests/typesafety/typescript/flag_documented.ts
// expect: passes

/** mode is "text" or "binary". */
function readLog(path: string, mode: string): number {
  const recordSize = mode === "text" ? 1 : 8;
  return path.length * recordSize;
}

export function tail(): number {
  return readLog("app.log", "binry");
}
